keyboards/niu_mini/keymaps/dyesub/keymap.c

```diff
@@ -0,0 +1,58 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a GPL2+ compatible license header at the beginning of the file, here?

Reply:
Sure, it's added now.